### PR TITLE
Bug 1048525 - Use SuggestClient from MC.

### PR DESCRIPTION
**Pull request by EricEdens**

### CODE REVIEW

File: app/src/main/java/org/mozilla/gecko/home/SuggestClient.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package org.mozilla.search.autocomplete;
+package org.mozilla.gecko.home;
```
Comment (leibovic) [inline app/src/main/java/org/mozilla/gecko/home/SuggestClient.java:5]: Do we need to move where the SuggestClient file lives in this repo? It should go over in an org.mozilla.gecko.home package, right?

Comment (EricEdens) [inline app/src/main/java/org/mozilla/gecko/home/SuggestClient.java:5]: Yeah, this patch handles the rename. It's not terribly clear, but if you look at the files changed (https://github.com/mozilla/fennec-search/pull/54/files), then this line indicates a rename:

/mozilla/search/autocomplete/SuggestClient.java → ...in/java/org/mozilla/gecko/home/SuggestClient.java

Comment (leibovic) [inline app/src/main/java/org/mozilla/gecko/home/SuggestClient.java:5]: Ah, I missed that line! Thanks for pointing that out.

Comment (EricEdens) [inline app/src/main/java/org/mozilla/gecko/home/SuggestClient.java:5]: Np :)

**Pull request merged**